ci/docker-build.sh

```diff
@@ -16,17 +15,8 @@ fi
 
 set -x
 
-# Build the base image which the image below can used.
-docker build \
-  --file ci/docker/Dockerfile.common \
-  --tag wasi-sdk-builder-base \
-  ci/docker
-
-# Build the container that is going to be used
-docker build \
-  --file ci/docker/Dockerfile.$1 \
-  --tag wasi-sdk-builder \
-  ci/docker
+# Build the Docker imager
+docker build --file ci/docker/Dockerfile --tag wasi-sdk-builder ci/docker
```

Comment:
I'm pretty sure you can remove `--file ci/docker/Dockerfile`.. it is documented as being the default value.